src/Sitegen.Tests/ProgramTest.cs

```diff
@@ -6,78 +6,88 @@
 
 namespace Sitegen.Tests
 {
-    public static class ReadConfig
+    public static class ProgramTest
```

Comment:
(Added missing outer class wrapping here, to make it clearer that these tests are testing the `Program` class. I also converted the assertions to use [Fluent Assertions](https://fluentassertions.com/) while I was at it. My friend @slovdahl would be proud of me, if he knew what it was. :wink:)